packages/components/src/internal/components/samples/SampleAliquotsGridPanel.spec.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import {
+    App,
+    LoadingSpinner,
+    LoadingState,
+    QueryInfo,
+    QueryModel,
+    SampleAliquotsGridPanel,
+    SchemaQuery,
+} from '../../../index';
+import { mountWithServerContext } from '../../testHelpers';
+
+describe('<SampleAliquotsGridPanel/>', () => {
+    const SAMPLE_TYPE_NAME = 'SampleTypeName';
+    const DEFAULT_CONTEXT = { user: App.TEST_USER_EDITOR };
+
+    const queryInfo = QueryInfo.create({ schemaName: 'samples', name: SAMPLE_TYPE_NAME, queryLabel: SAMPLE_TYPE_NAME });
```

Comment:
For instantiating a `QueryInfo` in our tests there is also a helper method called `makeTestQueryModel`. For your use case it would look something like:

```ts
queryModels: {
    model: makeTestQueryModel(SCHEMA_QUERY, new QueryInfo(), {}, [], 0, 'model'),
},
```

Reply:
Done (in patch)